test/jdk/java/nio/channels/Channels/TransferTo.java

```diff
@@ -139,16 +139,20 @@ public void testMoreThanTwoGB() throws IOException {
         Path targetFile = Files.createTempFile(null, null);
 
         // writing 3 GB of random bytes into source file
-        for (int i = 0; i < 3 * 1024; i++)
-            Files.write(sourceFile, createRandomBytes(1024 * 1024, 0), StandardOpenOption.APPEND);
+        int NUM_WRITES = 3 * 1024;
+        int BYTES_PER_WRITE = 1024 * 1024;
+        for (int i = 0; i < NUM_WRITES; i++)
+            Files.write(sourceFile, createRandomBytes(BYTES_PER_WRITE, 0), StandardOpenOption.APPEND);
 
         // performing actual transfer, effectively by multiple invocations of Filechannel.transferTo(FileChannel)
-        InputStream inputStream = Channels.newInputStream(FileChannel.open(sourceFile));
-        OutputStream outputStream = Channels.newOutputStream(FileChannel.open(targetFile, StandardOpenOption.WRITE));
-        long count = inputStream.transferTo(outputStream);
+        long count;
+        try (InputStream inputStream = Channels.newInputStream(FileChannel.open(sourceFile));
+        OutputStream outputStream = Channels.newOutputStream(FileChannel.open(targetFile, StandardOpenOption.WRITE))) {
+            count = inputStream.transferTo(outputStream);
+        }
 
         // comparing reported transferred bytes, must be 3 GB
-        assertEquals(count, 3L * 1024 * 1024 * 1024);
+        assertEquals(count, (long) NUM_WRITES * BYTES_PER_WRITE);
```

Comment:
This could also be a constant (similar to MAX_SIZE_INCR)

Reply:
Done in https://github.com/openjdk/jdk/pull/6093/commits/126e4877361ac7e3ab82101fee78d7d50c5733ac.